catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -269,15 +248,17 @@ private QueryResponse sourceFederate(List<Source> sources, final QueryRequest qu
                         logger.warn("Plugin stopped processing", e);
                     }
 
-                    finalQueryRequest = modifiedQueryRequest;
                     if (source instanceof CatalogProvider && SystemInfo.getSiteName()
                             .equals(source.getId())) {
-                        finalQueryRequest = getQueryRequestWithAdditionalFilters(
-                                modifiedQueryRequest);
+                        sourceQueryRequest =
```

Comment:
Ok. Can we make a ticket to move it out into it's own plugin. This validation logic was added here only because the other tags logic was here originally and it would be nice to clean all of it out of this class.